dev-php/promise/metadata.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>guillaumeseren@gmail.com</email>
+    <name>Guillaume Seren</name>
+  </maintainer>
+  <maintainer type="project">
+    <email>proxy-maint@gentoo.org</email>
+    <name>Proxy Maintainers</name>
+  </maintainer>
+  <maintainer type="project">
+    <email>php-bugs@gentoo.org</email>
+  </maintainer>
```

Comment:
And we can't add this project without their consent either :\

Reply:
It's a new direct dependency, that's why the current maintainers are added.

Reply:
I am still working on this, currently busy. Hope to get to this in this week.

Reply:
@Whissi ping.

Reply:
Sorry for the delay, had to get phpunit problems out of the way first. Looking forward for the coming week, but PHP 8 will be first.